packages/core/src/app.ts

```diff
@@ -1520,6 +1520,8 @@ export class App {
             attachments?: AttachmentInfo[];
             auditResults?: AuditResult[];
             lastAudited?: Date;
+            expiresBy?: Date;
```

Comment:
*Very* minor nitpick: `expiryDate` would be slightly more consistent with `expiryFrequency`.

Reply:
True, but I don't like how it reads or sounds. :sweat_smile: